src/components/event/LocationSelect.tsx

```diff
@@ -43,21 +44,31 @@ interface PlaceType {
 
 export default function GoogleMaps() {
   const classes = useStyles();
+  //TODO add to .env
+  const gmailApiKey = 'AIzaSyDEghbBwMptrjhaBo16TMRL3z4pK3UN438';
```

Comment:
Please rename it to `gMapsApiKey` or `googleMapsApiKey`. Also, extract the value into an environment variable.